2023-06-17 21:40:47,704 INFO    StreamThr :290 [internal.py:wandb_internal():86] W&B internal server running at pid: 290, started at: 2023-06-17 21:40:47.703571
2023-06-17 21:40:47,706 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status
2023-06-17 21:40:47,708 INFO    WriterThread:290 [datastore.py:open_for_write():85] open: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/run-ln49k86n.wandb
2023-06-17 21:40:47,716 DEBUG   SenderThread:290 [sender.py:send():369] send: header
2023-06-17 21:40:47,716 DEBUG   SenderThread:290 [sender.py:send():369] send: run
2023-06-17 21:40:52,715 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:40:57,717 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:02,719 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:07,721 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:12,724 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:17,727 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:21,841 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: check_version
2023-06-17 21:41:21,841 INFO    SenderThread:290 [dir_watcher.py:__init__():211] watching files in: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files
2023-06-17 21:41:21,841 INFO    SenderThread:290 [sender.py:_start_run_threads():1100] run started: ln49k86n with start time 1687009247.704844
2023-06-17 21:41:21,842 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: summary_record
2023-06-17 21:41:21,842 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:21,844 INFO    SenderThread:290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-06-17 21:41:21,845 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: check_version
2023-06-17 21:41:22,505 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: run_start
2023-06-17 21:41:22,546 DEBUG   HandlerThread:290 [system_info.py:__init__():31] System info init
2023-06-17 21:41:22,546 DEBUG   HandlerThread:290 [system_info.py:__init__():46] System info init done
2023-06-17 21:41:22,546 INFO    HandlerThread:290 [system_monitor.py:start():181] Starting system monitor
2023-06-17 21:41:22,547 INFO    SystemMonitor:290 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-06-17 21:41:22,547 INFO    HandlerThread:290 [system_monitor.py:probe():201] Collecting system info
2023-06-17 21:41:22,547 INFO    SystemMonitor:290 [interfaces.py:start():190] Started cpu monitoring
2023-06-17 21:41:22,549 INFO    SystemMonitor:290 [interfaces.py:start():190] Started disk monitoring
2023-06-17 21:41:22,549 INFO    SystemMonitor:290 [interfaces.py:start():190] Started gpu monitoring
2023-06-17 21:41:22,550 INFO    SystemMonitor:290 [interfaces.py:start():190] Started memory monitoring
2023-06-17 21:41:22,551 INFO    SystemMonitor:290 [interfaces.py:start():190] Started network monitoring
2023-06-17 21:41:22,564 DEBUG   HandlerThread:290 [system_info.py:probe():195] Probing system
2023-06-17 21:41:22,596 DEBUG   HandlerThread:290 [gitlib.py:_init_repo():53] git repository is invalid
2023-06-17 21:41:22,596 DEBUG   HandlerThread:290 [system_info.py:probe():240] Probing system done
2023-06-17 21:41:22,597 DEBUG   HandlerThread:290 [system_monitor.py:probe():210] {'os': 'Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-06-17T13:41:22.565287', 'startedAt': '2023-06-17T13:40:47.661797', 'docker': None, 'cuda': None, 'args': ('wandb', 'login'), 'state': 'running', 'program': '/mnt/d/PROJECTION/TianChi/STS-2D/train.py', 'codePath': 'train.py', 'host': 'DESKTOP-LT0ASMB', 'username': 'zy', 'executable': '/home/zy/anaconda3/envs/kaggle_torch/bin/python3', 'cpu_count': 3, 'cpu_count_logical': 5, 'cpu_freq': {'current': 3191.997, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 3191.997, 'min': 0.0, 'max': 0.0}, {'current': 3191.997, 'min': 0.0, 'max': 0.0}, {'current': 3191.997, 'min': 0.0, 'max': 0.0}, {'current': 3191.997, 'min': 0.0, 'max': 0.0}, {'current': 3191.997, 'min': 0.0, 'max': 0.0}], 'disk': {'total': 250.98248672485352, 'used': 49.27463912963867}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 24.93661880493164}}
2023-06-17 21:41:22,597 INFO    HandlerThread:290 [system_monitor.py:probe():211] Finished collecting system info
2023-06-17 21:41:22,597 INFO    HandlerThread:290 [system_monitor.py:probe():214] Publishing system info
2023-06-17 21:41:22,597 DEBUG   HandlerThread:290 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-06-17 21:41:22,600 DEBUG   HandlerThread:290 [system_info.py:_save_pip():67] Saving pip packages done
2023-06-17 21:41:22,600 DEBUG   HandlerThread:290 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-06-17 21:41:22,625 ERROR   HandlerThread:290 [system_info.py:_save_conda():85] Error saving conda packages: [Errno 2] No such file or directory: 'conda'
Traceback (most recent call last):
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/wandb/sdk/internal/system/system_info.py", line 81, in _save_conda
    subprocess.call(
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda'
2023-06-17 21:41:22,626 DEBUG   HandlerThread:290 [system_info.py:_save_conda():86] Saving conda packages done
2023-06-17 21:41:22,629 INFO    HandlerThread:290 [system_monitor.py:probe():216] Finished publishing system info
2023-06-17 21:41:22,635 DEBUG   SenderThread:290 [sender.py:send():369] send: files
2023-06-17 21:41:22,635 INFO    SenderThread:290 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-06-17 21:41:22,641 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: stop_status
2023-06-17 21:41:22,642 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: stop_status
2023-06-17 21:41:22,850 INFO    Thread-12 :290 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/conda-environment.yaml
2023-06-17 21:41:22,851 INFO    Thread-12 :290 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-summary.json
2023-06-17 21:41:22,854 INFO    Thread-12 :290 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/requirements.txt
2023-06-17 21:41:22,855 INFO    Thread-12 :290 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-metadata.json
2023-06-17 21:41:22,916 DEBUG   SenderThread:290 [sender.py:send():369] send: telemetry
2023-06-17 21:41:23,855 INFO    Thread-12 :290 [dir_watcher.py:_on_file_created():272] file/dir created: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:25,866 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:26,872 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:26,924 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:26,929 DEBUG   SenderThread:290 [sender.py:send():369] send: exit
2023-06-17 21:41:26,929 INFO    SenderThread:290 [sender.py:send_exit():574] handling exit code: 1
2023-06-17 21:41:26,930 INFO    SenderThread:290 [sender.py:send_exit():576] handling runtime: 4
2023-06-17 21:41:26,932 INFO    SenderThread:290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-06-17 21:41:26,933 INFO    SenderThread:290 [sender.py:send_exit():582] send defer
2023-06-17 21:41:26,934 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,934 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 0
2023-06-17 21:41:26,935 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,935 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 0
2023-06-17 21:41:26,935 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 1
2023-06-17 21:41:26,936 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,936 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 1
2023-06-17 21:41:26,936 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,937 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 1
2023-06-17 21:41:26,937 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 2
2023-06-17 21:41:26,937 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,938 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 2
2023-06-17 21:41:26,938 INFO    HandlerThread:290 [system_monitor.py:finish():190] Stopping system monitor
2023-06-17 21:41:26,938 INFO    HandlerThread:290 [interfaces.py:finish():202] Joined cpu monitor
2023-06-17 21:41:26,939 INFO    HandlerThread:290 [interfaces.py:finish():202] Joined disk monitor
2023-06-17 21:41:26,939 DEBUG   SystemMonitor:290 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-06-17 21:41:26,939 DEBUG   SystemMonitor:290 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-06-17 21:41:26,940 DEBUG   SystemMonitor:290 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-06-17 21:41:26,951 INFO    HandlerThread:290 [interfaces.py:finish():202] Joined gpu monitor
2023-06-17 21:41:26,952 INFO    HandlerThread:290 [interfaces.py:finish():202] Joined memory monitor
2023-06-17 21:41:26,952 INFO    HandlerThread:290 [interfaces.py:finish():202] Joined network monitor
2023-06-17 21:41:26,952 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,953 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 2
2023-06-17 21:41:26,953 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 3
2023-06-17 21:41:26,953 DEBUG   SenderThread:290 [sender.py:send():369] send: stats
2023-06-17 21:41:26,953 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,954 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 3
2023-06-17 21:41:26,954 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,955 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 3
2023-06-17 21:41:26,955 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 4
2023-06-17 21:41:26,955 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,956 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 4
2023-06-17 21:41:26,956 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,956 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 4
2023-06-17 21:41:26,956 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 5
2023-06-17 21:41:26,957 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,957 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 5
2023-06-17 21:41:26,958 DEBUG   SenderThread:290 [sender.py:send():369] send: summary
2023-06-17 21:41:26,960 INFO    SenderThread:290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-06-17 21:41:26,961 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,961 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 5
2023-06-17 21:41:26,961 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 6
2023-06-17 21:41:26,961 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:26,961 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 6
2023-06-17 21:41:26,962 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:26,962 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 6
2023-06-17 21:41:26,964 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:27,272 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 7
2023-06-17 21:41:27,273 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:27,273 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 7
2023-06-17 21:41:27,273 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:27,274 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 7
2023-06-17 21:41:27,896 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-summary.json
2023-06-17 21:41:27,898 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:27,898 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/config.yaml
2023-06-17 21:41:27,930 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: poll_exit
2023-06-17 21:41:28,922 INFO    Thread-12 :290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:30,598 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 8
2023-06-17 21:41:30,600 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: poll_exit
2023-06-17 21:41:30,601 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:30,603 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 8
2023-06-17 21:41:30,605 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:30,606 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 8
2023-06-17 21:41:30,608 INFO    SenderThread:290 [job_builder.py:build():232] Attempting to build job artifact
2023-06-17 21:41:30,631 INFO    SenderThread:290 [job_builder.py:build():266] no source found
2023-06-17 21:41:30,632 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 9
2023-06-17 21:41:30,633 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:30,634 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 9
2023-06-17 21:41:30,637 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:30,638 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 9
2023-06-17 21:41:30,639 INFO    SenderThread:290 [dir_watcher.py:finish():359] shutting down directory watcher
2023-06-17 21:41:30,968 INFO    SenderThread:290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:31,005 INFO    SenderThread:290 [dir_watcher.py:finish():389] scan: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files
2023-06-17 21:41:31,008 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/conda-environment.yaml conda-environment.yaml
2023-06-17 21:41:31,012 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/config.yaml config.yaml
2023-06-17 21:41:31,015 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log output.log
2023-06-17 21:41:31,020 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/requirements.txt requirements.txt
2023-06-17 21:41:31,024 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-metadata.json wandb-metadata.json
2023-06-17 21:41:31,025 INFO    SenderThread:290 [dir_watcher.py:finish():403] scan save: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-summary.json wandb-summary.json
2023-06-17 21:41:31,029 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 10
2023-06-17 21:41:31,029 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:31,029 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 10
2023-06-17 21:41:31,030 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:31,032 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 10
2023-06-17 21:41:31,032 INFO    SenderThread:290 [file_pusher.py:finish():159] shutting down file pusher
2023-06-17 21:41:32,032 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:32,932 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:33,199 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/wandb-metadata.json?Expires=1687095682&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=ZnfuDj27lbdOMG2hF4jXscHdXPbxm7rpyY6G1iKhaax8WzscWu7KNguSqinRM9VJXWGSDYlJxAC4WfwpDpzfyQMUFyWhh4NodG8mU%2FCuNu91laCKQGTj1lVcpBaDRDG6DMwZB0emzemmJ5CbwtTHRAYK8wGfjEUGt%2BQNyCFbrn5PrHlkTi7znJIMpWysibyTcinEACaENn00R8tfa70LRS%2BVkPVuiMdaGBO7Cn6ltm1Ft4MFgDAMbAM1J36drjCEz9JQoMm4gAHpwgVxoe90fkU0ATRXMxhm8KNYmUUYwpvYvZ4xhLGTT0Ce2dwGgOQ1Ejh0jTqTuurQTRGPKnpUeg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/wandb-metadata.json?Expires=1687095682&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=ZnfuDj27lbdOMG2hF4jXscHdXPbxm7rpyY6G1iKhaax8WzscWu7KNguSqinRM9VJXWGSDYlJxAC4WfwpDpzfyQMUFyWhh4NodG8mU%2FCuNu91laCKQGTj1lVcpBaDRDG6DMwZB0emzemmJ5CbwtTHRAYK8wGfjEUGt%2BQNyCFbrn5PrHlkTi7znJIMpWysibyTcinEACaENn00R8tfa70LRS%2BVkPVuiMdaGBO7Cn6ltm1Ft4MFgDAMbAM1J36drjCEz9JQoMm4gAHpwgVxoe90fkU0ATRXMxhm8KNYmUUYwpvYvZ4xhLGTT0Ce2dwGgOQ1Ejh0jTqTuurQTRGPKnpUeg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb188288c40>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023-06-17 21:41:33,201 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1547'}
2023-06-17 21:41:33,202 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2196] upload_file response body: 
2023-06-17 21:41:33,854 WARNING FileStreamThread:290 [file_stream.py:request_with_retry():665] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/zhuangy/STS-2D/ln49k86n/file_stream (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb188b4d160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')). func: <bound method Session.post of <requests.sessions.Session object at 0x7fb18ab96340>>, args: ('https://api.wandb.ai/files/zhuangy/STS-2D/ln49k86n/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{}']}}, 'dropped': 0}}
2023-06-17 21:41:37,034 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:37,936 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:41,318 ERROR   wandb-upload_1:290 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/config.yaml?Expires=1687095690&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=AWZxMK3wR2sEPx412eAfPexWXSiMobV6jKrNj25DiiBBCyNceqgR83RNeYY%2BHVixo3K60ECGC%2BgotmB%2BBVgV521mWeFvOmjslaZHvD8L%2BBbRPhAeSGjb%2B3iQRSqs3vcd0SLDsT3Sqib9JXs%2FZkMZ%2Bnr6r9sxfq5fndfQrjW7D9Jflu39PDnvGVhEwur1UmWfcczgv2RU0BPnZGusVCItdrDx6K5c8rtTOBqdJNa%2F%2BzPnZH6crPH4Mynn3S3tX8tkC0Fvf0mlFjdV94ebmRxRIuQHImJApoHr2Qc1Oevhit%2FIeVTxt%2FywXzya7HjwOJY8zy34FtgPaetWyOaIMOi5lA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/config.yaml?Expires=1687095690&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=AWZxMK3wR2sEPx412eAfPexWXSiMobV6jKrNj25DiiBBCyNceqgR83RNeYY%2BHVixo3K60ECGC%2BgotmB%2BBVgV521mWeFvOmjslaZHvD8L%2BBbRPhAeSGjb%2B3iQRSqs3vcd0SLDsT3Sqib9JXs%2FZkMZ%2Bnr6r9sxfq5fndfQrjW7D9Jflu39PDnvGVhEwur1UmWfcczgv2RU0BPnZGusVCItdrDx6K5c8rtTOBqdJNa%2F%2BzPnZH6crPH4Mynn3S3tX8tkC0Fvf0mlFjdV94ebmRxRIuQHImJApoHr2Qc1Oevhit%2FIeVTxt%2FywXzya7HjwOJY8zy34FtgPaetWyOaIMOi5lA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb1882b61f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023-06-17 21:41:41,320 ERROR   wandb-upload_1:290 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '955'}
2023-06-17 21:41:41,321 ERROR   wandb-upload_1:290 [internal_api.py:upload_file():2196] upload_file response body: 
2023-06-17 21:41:41,327 ERROR   wandb-upload_2:290 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/output.log?Expires=1687095690&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=gLGSLjk4VLWeESx7w9VYS622pZ5pFDFm78c1qahkBIDeSdrx2Jlg%2B5Pwqabu5tjc7NHoGJ8e1IoB%2BMIk5BAfIqQa0EZPETKOUqEGNywTJ1qjfKC35Kpgngo6xFvQBTO%2FLaGEFlQ9mZXxn1ZsXvtjkj8jBtx5RiQzvXC%2BC6xYhfF%2B8s4dopc3hxDNvqEvxJ%2B5eK8hrJLEVmtZ4K2L9Ov8Tx6ay0nNTtHXKuwd2oCMZvLixPHuXCsHrBqPy1lAih64L52RKiS5xltILQMWBrRp2azsAqwex4OuAIFcuk3GjwqT2yv3zxKfi5DYKyqpMPr9EsjNCMrNuoQGTI40CbgutQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/output.log?Expires=1687095690&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=gLGSLjk4VLWeESx7w9VYS622pZ5pFDFm78c1qahkBIDeSdrx2Jlg%2B5Pwqabu5tjc7NHoGJ8e1IoB%2BMIk5BAfIqQa0EZPETKOUqEGNywTJ1qjfKC35Kpgngo6xFvQBTO%2FLaGEFlQ9mZXxn1ZsXvtjkj8jBtx5RiQzvXC%2BC6xYhfF%2B8s4dopc3hxDNvqEvxJ%2B5eK8hrJLEVmtZ4K2L9Ov8Tx6ay0nNTtHXKuwd2oCMZvLixPHuXCsHrBqPy1lAih64L52RKiS5xltILQMWBrRp2azsAqwex4OuAIFcuk3GjwqT2yv3zxKfi5DYKyqpMPr9EsjNCMrNuoQGTI40CbgutQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb18826f850>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023-06-17 21:41:41,328 ERROR   wandb-upload_2:290 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1029'}
2023-06-17 21:41:41,329 ERROR   wandb-upload_2:290 [internal_api.py:upload_file():2196] upload_file response body: 
2023-06-17 21:41:42,036 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:42,939 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: keepalive
2023-06-17 21:41:44,294 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/wandb-metadata.json?Expires=1687095682&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=ZnfuDj27lbdOMG2hF4jXscHdXPbxm7rpyY6G1iKhaax8WzscWu7KNguSqinRM9VJXWGSDYlJxAC4WfwpDpzfyQMUFyWhh4NodG8mU%2FCuNu91laCKQGTj1lVcpBaDRDG6DMwZB0emzemmJ5CbwtTHRAYK8wGfjEUGt%2BQNyCFbrn5PrHlkTi7znJIMpWysibyTcinEACaENn00R8tfa70LRS%2BVkPVuiMdaGBO7Cn6ltm1Ft4MFgDAMbAM1J36drjCEz9JQoMm4gAHpwgVxoe90fkU0ATRXMxhm8KNYmUUYwpvYvZ4xhLGTT0Ce2dwGgOQ1Ejh0jTqTuurQTRGPKnpUeg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/zhuangy/STS-2D/ln49k86n/wandb-metadata.json?Expires=1687095682&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=ZnfuDj27lbdOMG2hF4jXscHdXPbxm7rpyY6G1iKhaax8WzscWu7KNguSqinRM9VJXWGSDYlJxAC4WfwpDpzfyQMUFyWhh4NodG8mU%2FCuNu91laCKQGTj1lVcpBaDRDG6DMwZB0emzemmJ5CbwtTHRAYK8wGfjEUGt%2BQNyCFbrn5PrHlkTi7znJIMpWysibyTcinEACaENn00R8tfa70LRS%2BVkPVuiMdaGBO7Cn6ltm1Ft4MFgDAMbAM1J36drjCEz9JQoMm4gAHpwgVxoe90fkU0ATRXMxhm8KNYmUUYwpvYvZ4xhLGTT0Ce2dwGgOQ1Ejh0jTqTuurQTRGPKnpUeg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb18826fb50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023-06-17 21:41:44,295 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1547'}
2023-06-17 21:41:44,295 ERROR   wandb-upload_0:290 [internal_api.py:upload_file():2196] upload_file response body: 
2023-06-17 21:41:46,412 INFO    wandb-upload_1:290 [upload_job.py:push():133] Uploaded file /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/config.yaml
2023-06-17 21:41:46,957 INFO    wandb-upload_2:290 [upload_job.py:push():133] Uploaded file /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/output.log
2023-06-17 21:41:47,038 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: status_report
2023-06-17 21:41:47,138 INFO    wandb-upload_4:290 [upload_job.py:push():133] Uploaded file /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/wandb-summary.json
2023-06-17 21:41:47,256 INFO    wandb-upload_3:290 [upload_job.py:push():133] Uploaded file /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/files/requirements.txt
2023-06-17 21:41:47,353 INFO    wandb-upload_0:290 [upload_job.py:push():133] Uploaded file /tmp/tmpx6bn76qcwandb/ycwvfkxj-wandb-metadata.json
2023-06-17 21:41:47,554 INFO    Thread-11 :290 [sender.py:transition_state():602] send defer: 11
2023-06-17 21:41:47,559 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:47,564 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 11
2023-06-17 21:41:47,570 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:47,582 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 11
2023-06-17 21:41:47,584 INFO    SenderThread:290 [file_pusher.py:join():164] waiting for file pusher
2023-06-17 21:41:47,585 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 12
2023-06-17 21:41:47,598 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:47,598 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 12
2023-06-17 21:41:47,598 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:47,599 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 12
2023-06-17 21:41:47,880 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 13
2023-06-17 21:41:47,880 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:47,881 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 13
2023-06-17 21:41:47,881 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:47,882 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 13
2023-06-17 21:41:47,882 INFO    SenderThread:290 [sender.py:transition_state():602] send defer: 14
2023-06-17 21:41:47,883 DEBUG   SenderThread:290 [sender.py:send():369] send: final
2023-06-17 21:41:47,883 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: defer
2023-06-17 21:41:47,883 INFO    HandlerThread:290 [handler.py:handle_request_defer():170] handle defer: 14
2023-06-17 21:41:47,883 DEBUG   SenderThread:290 [sender.py:send():369] send: footer
2023-06-17 21:41:47,884 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: defer
2023-06-17 21:41:47,884 INFO    SenderThread:290 [sender.py:send_request_defer():598] handle sender defer: 14
2023-06-17 21:41:47,885 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: poll_exit
2023-06-17 21:41:47,885 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: server_info
2023-06-17 21:41:47,885 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: poll_exit
2023-06-17 21:41:47,886 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: get_summary
2023-06-17 21:41:47,886 DEBUG   SenderThread:290 [sender.py:send_request():396] send_request: server_info
2023-06-17 21:41:47,886 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: sampled_history
2023-06-17 21:41:48,436 INFO    MainThread:290 [wandb_run.py:_footer_history_summary_info():3467] rendering history
2023-06-17 21:41:48,437 INFO    MainThread:290 [wandb_run.py:_footer_history_summary_info():3499] rendering summary
2023-06-17 21:41:48,439 INFO    MainThread:290 [wandb_run.py:_footer_sync_info():3426] logging synced files
2023-06-17 21:41:48,441 DEBUG   HandlerThread:290 [handler.py:handle_request():144] handle_request: shutdown
2023-06-17 21:41:48,442 INFO    HandlerThread:290 [handler.py:finish():854] shutting down handler
2023-06-17 21:41:48,886 INFO    WriterThread:290 [datastore.py:close():298] close: /mnt/d/PROJECTION/TianChi/STS-2D/wandb/run-20230617_214047-ln49k86n/run-ln49k86n.wandb
2023-06-17 21:41:49,436 INFO    SenderThread:290 [sender.py:finish():1526] shutting down sender
2023-06-17 21:41:49,436 INFO    SenderThread:290 [file_pusher.py:finish():159] shutting down file pusher
2023-06-17 21:41:49,436 INFO    SenderThread:290 [file_pusher.py:join():164] waiting for file pusher
